_layouts/post.html

```diff
@@ -82,14 +82,17 @@ <h3 class="page-title">Want to read more about {{ page.category }}? Check out th
 <section class="comments">
     <div class="grid">
         <div class="grid-1-1">
+            {% if page.disablecomments %}
+            <div id="disqus_thread">Comments have been disabled for this post</div>
+            {% else %}
             <div id="disqus_thread"></div>
+            {% endif %}
         </div>
     </div>
 </section>
 
-{% if page.disablecomments %}
-Comments for this post have been disabled
-{% else %}
+
+{% if page.disablecomments == nil or page.disablecomments == false %}
```

Comment:
OK, I've changed it to `unless`.  I tend to use `if` because it's easier for non-ruby people to read.